api/v1/testkube.yaml

```diff
@@ -117,6 +117,35 @@ paths:
                 items:
                   $ref: "#/components/schemas/Problem"
 
+  /test-suite-with-executions:
+    get:
+      tags:
+        - test-suites
+        - api
+      summary: "Get all test suite with executions"
+      description: "Returns array of test suite with executions"
+      operationId: listTestSuiteWithExecutions
+      parameters:
+        - $ref: "#/components/parameters/Selector"
+        - $ref: "#/components/parameters/TextSearch"
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/TestSuiteWithExecution"
```

Comment:
is TestSuiteWithExecution defined in the api schema?

Reply:
yes, it was done on previous pr

Reply:
Now i see it was defined before